stix_shifter_modules/rhacs/configuration/lang_en.json

```diff
@@ -13,15 +13,15 @@
       "description": "More details on the data source setting can be found in the specified link"
     },
     "selfSignedCert": {
-      "label": "RHACS connection certificate",
-      "description": "Optional self-signed SSL certificate or CA content (root and intermediate)"
-    }
+      "label": "PEM Formatted SSL certificate(s)",
+      "description": "Provide a self-signed or CA-signed certificate to securely communicate with the data source."
+  }
   },
   "configuration": {
     "auth": {
       "token": {
-        "label": "API token",
-        "description": "Token with readonly access to the alert API"
```

Comment:
we need to include this piece of information about `readonly access to the alert API` in the description